src/utils/index.ts

```diff
@@ -108,6 +108,18 @@ export async function promptForWalletConnectIdWithLogs(
     );
 }
 
+export async function promptForStytchWithLogs(
+    stytchPublicToken: string | undefined
+): Promise<string | undefined> {
+    return await promptForKeyWithLogs(
+        { key: stytchPublicToken, inputMessage: "Stytch Public Token:" },
+        [
+            "Please provide the Stytch Public Token for your project.",
+            "To skip and use the default test client ID, press enter.",
+        ]
+    );
+}
+
```

Comment:
Where can I find my Stytch public token? Can you add that detail there?
Image with an example:

![image](https://github.com/user-attachments/assets/3695b9c7-f50e-45a4-827f-b275fbf47f4c)